util/stopper.go

```diff
@@ -162,13 +160,20 @@ func (s *Stopper) SetStopped() {
 	}
 }
 
-// Quiesce moves the stopper to state draining, waits until all tasks
-// complete, then moves back to non-draining state. This is used from
-// unittests.
+// Quiesce moves the stopper to state draining and waits until all
+// tasks complete. This is used from Stop() and unittests.
 func (s *Stopper) Quiesce() {
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	s.draining = 1
-	s.drain.Wait()
-	s.draining = 0
+	s.draining = true
+	for s.numTasks > 0 {
+		s.drain.Wait()
+	}
+}
+
+// Unquiesce moves back to non-draining state. This is used from unittests.
+func (s *Stopper) Unquiesce() {
```

Comment:
looks like this method is only in `TestStopperUnquiesce`, which only tests this method. Why not remove it?